Makefile

```diff
@@ -117,9 +115,7 @@ update-mockgen:
 
 update-proto-plugins:
 	@printf $(COLOR) "Install/update proto plugins..."
-	@go install github.com/temporalio/gogo-protobuf/protoc-gen-gogoslick@master
-# This to download sources of gogo-protobuf which are required to build proto files.
-	@GO111MODULE=off go get github.com/temporalio/gogo-protobuf/protoc-gen-gogoslick
+	@(cd build && go install github.com/temporalio/gogo-protobuf/protoc-gen-gogoslick)
 	@go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
```

Comment:
should this be in `build` too? how about mockgen and all the stuff under update-checkers?

Reply:
All tools should be there. I will do it in separate PR.